variables.tf

```diff
@@ -25,3 +25,14 @@ variable "lock_table_read_capacity" {
 variable "lock_table_write_capacity" {
   default = 1
 }
+
+variable "sse_algorithm" {
+  default = "aws:kms"
+  description = "The server-side encryption algorithm to use for the tf state s3 bucket. Valid values are AES256 and aws:kms"
+}
+
+variable "kms_key_id" {
+  # Default to absent/blank to use the default aws/s3 aws kms master key
+  default = ""
```

Comment:
Need to make sure that this *actually* works. There is a chance that this will actually try to use a literal blank string as the key id, which will likely fail on the bucket creation/modification. Even if it doesn't fail, we need to make sure that this default setting `""` results in the use of the `aws/s3 kms master key`.